src/generators/app.ts

```diff
@@ -233,9 +239,9 @@ class App extends Generator {
     this.pjson.repository = this.answers.github ? `${this.answers.github.user}/${this.answers.github.repo}` : defaults.repository
     this.pjson.scripts.posttest = 'yarn run lint'
     // this.pjson.scripts.precommit = 'yarn run lint'
-    if (this.ts && this.mocha) {
+    if (this.ts && this.tslint && this.mocha) {
       this.pjson.scripts.lint = 'tsc -p test --noEmit && tslint -p test -t stylish'
```

Comment:
if we don't select tslint, we should still run `tsc -p test`. The way this is written right now it would fall back to `eslint` which doesn't come with typescript setups

Reply:
Oops, good catch!